CHANGELOG-PENDING.md

```diff
@@ -14,7 +14,7 @@ Month, DD, YYYY
 
 ### IMPROVEMENTS
 
-- [go package] (Link to PR) Description @username
+- [indexer] Implement block and transaction indexing, enable TxSearch RPC endpoint @mattdf
```

Comment:
```suggestion
- [indexer] [Implement block and transaction indexing, enable TxSearch RPC endpoint #202](https://github.com/celestiaorg/optimint/pull/202) @mattdf
```

Reply:
i had it this way initially but josh told me to remove the PR link because it implies the changelog is changing in a separate PR? :sob:

Reply:
OK, I think we desperately need contrib guidelines to have this somewhat consistent across repos. 
e.g. looking into https://github.com/celestiaorg/celestia-node/blob/main/CHANGELOG-PENDING.md
it seems node follows the other pattern.